modern/src/Theme.js

```diff
@@ -0,0 +1,50 @@
+import { createMuiTheme } from '@material-ui/core/styles';
+
+const traccarPurple = '#333366';
+const traccarGreen = '#4CAF50';
+const traccarWhite = '#FFF';
+
+export default createMuiTheme({
+  palette: {  
+    common: {
+      purple: traccarPurple,
+      green: traccarGreen
+    },
+    primary: {
+      main: traccarPurple
+    },
+    secondary: {
+      main: traccarGreen,
+      contrastText: traccarWhite
+    }
+  },
+  overrides: {
+    MuiFormControl: {
+      root: {
+        height: '42px',
+      }
+    },    
+    MuiFilledInput: {
```

Comment:
We will need to style the input as well. Chrome autofill shows a background which looks awkward without the input style.

Here's what I tried that works

```javascript
      input:{
        height: '42px',
        borderRadius: '4px',
        paddingTop: '10px',
        boxSizing: 'border-box'
      },
```

Reply:
Done.
I have also added box-shadow which I found online and working fine. Without box-shadow, there is a background-color different from our theme color.